pymola/ast.py

```diff
@@ -211,7 +211,7 @@ def __init__(self, **kwargs):
         self.max = Primary(value=None)  # type: Union[Expression, Primary, ComponentRef, Array]
         self.nominal = Primary(value=None)  # type: Union[Expression, Primary, ComponentRef, Array]
         self.value = Primary(value=None)  # type: Union[Expression, Primary, ComponentRef, Array]
-        self.fixed = False  # type: bool
+        self.fixed = Primary(value=False)  # type: Primary
```

Comment:
what is the thinking here? I  would think that making it a primary is overkill if it is always going to be type bool?